README.md

```diff
@@ -5,6 +5,15 @@
 # Secure IO
 ## Go implementation of the Data At Rest Encryption (DARE) format.
 
+***Abstract:** DARE is an authenticated encryption scheme to en/decrypt **and** 
+authenticate/verify arbitrary long data data streams using a **constant** amount of space. Classical 
```

Comment:
s/data data/data

Also, I think saying **memory** instead of **space** is better.